bin/hdfcoinc/pycbc_multiifo_sngls_findtrigs

```diff
@@ -31,59 +35,126 @@ parser.add_argument("--statistic-keywords", nargs='*',
                          "the statistic class when it is initialized. Should "
                          "be given in format --statistic-keywords "
                          "KWARG1:VALUE1 KWARG2:VALUE2 KWARG3:VALUE3 ...")
-parser.add_argument("--template-fraction-range", default="0/1",
-                    help="Optional, analyze only part of template bank. Format"
-                         " PART/NUM_PARTS")
-parser.add_argument("--randomize-template-order", action="store_true",
-                    help="Random shuffle templates with fixed seed "
-                         "before selecting range to analyze")
-parser.add_argument("--cluster-window", type=float,
-                    help="Optional, window size in seconds to cluster "
-                         "over the bank")
+parser.add_argument('--trigger-snr-cut',  type=float,
+                    help='Only consider triggers above the given SNR.')
+parser.add_argument('--cluster-window', type=float,
+                    help='Window (seconds) during which to keep the trigger '
+                         'with the loudest statistic value. '
+                         'Default=do not cluster')
+parser.add_argument('--reduced-chisq-cut', type=float,
+                    help='Only consider triggers below given reduced '
+                         'chisquared.')
+parser.add_argument('--inj-filter-file',
+                    help="If analysing injections, use a HDF_INJFIND file to "
+                         "select the trigger ids of found injections")
+parser.add_argument('--inj-filter-ifar', default=0, type=float,
+                    help="If analysing injections, set a minimum IFAR for "
+                         "deciding if an injection is found. Default=0")
```

Comment:
This code runs independent of injection information currently. I can't really parse what this file is used for? As the output of this code is needed to produce the HDF_INJFIND, this seems to introduce a cyclical dependency